akka-http-core/src/main/scala/akka/http/javadsl/Http.scala

```diff
@@ -505,7 +505,7 @@ class Http(system: ExtendedActorSystem) extends akka.actor.Extension {
   /**
    * Same as [[cachedHostConnectionPool]] but with HTTPS encryption.
    *
-   * The given [[ConnectionContext]] will be used for encryption on the connection.
+   * The current default client-side [[ConnectionContext]] will be used for encryption on the connection.
```

Comment:
That's not quite correct. `ConnectHttp` can contain a custom context which will then be used. Maybe change the comment to

> When a [[HttpConnectionContext]] is defined in [[ConnectHttp]] this will be used, otherwise the default client-side context will be used.

Same below.

Reply:
I went ahead and applied the suggested doc change.